include/swift/ABI/MetadataValues.h

```diff
@@ -542,19 +542,22 @@ class ExistentialTypeFlags {
 enum class FunctionMetadataConvention: uint8_t {
   Swift = 0,
   Block = 1,
-  Thin = 2,
+  Thin  = 2,
```

Comment:
This seems unnecessary.  It's important to line up bitmask constants, not all constants.

Reply:
Ok, let me remove that indent.